src/project/types/website/website.ts

```diff
@@ -207,8 +208,12 @@ export const websiteProjectType: ProjectType = {
         extras.metadataOverride[kFieldCategories] = asArray(
           format.metadata[kFieldCategories],
         ).map(
-          (category) =>
-            pandocNativeStr(category as string).mappedString().value,
+          (category) => {
+            const strCategory: string = typeof category === "string"
+              ? category
+              : category.toString();
+            return pandocNativeStr(strCategory).mappedString().value;
+          },
```

Comment:
I now understand what was the issue ! 

Thanks for having DM-ed me about this so that I learn better about this. 

Thanks.